crates/common_utils/src/consts.rs

```diff
@@ -149,3 +154,69 @@ pub const ROLE_ID_ORGANIZATION_ADMIN: &str = "org_admin";
 pub const ROLE_ID_INTERNAL_VIEW_ONLY_USER: &str = "internal_view_only";
 /// Role ID for Internal Admin
 pub const ROLE_ID_INTERNAL_ADMIN: &str = "internal_admin";
+
+/// Regex for Identifying Card Network
+pub const CARD_NETWORK_DATA: Lazy<HashMap<common_enums::CardNetwork, CardNetworkPattern>> =
+    Lazy::new(|| {
+        let mut map = HashMap::new();
+        map.insert(common_enums::CardNetwork::Maestro, CardNetworkPattern {
+        regex:  Regex::new(r"^(5018|5081|5044|504681|504993|5020|502260|5038|603845|603123|6304|6759|676[1-3]|6220|504834|504817|504645|504775|600206|627741)[0-9]{0,15}$").ok(), 
```

Comment:
isn't it unsafe to ignore these errors?

Reply:
Yes, this will be unsafe. Fixed it. Now we are throwing an internal server error